modin/experimental/cloud/rayscale.py

```diff
@@ -148,7 +149,9 @@ def _conda_requirements(self):
 
         reqs.extend(self._get_python_version())
 
-        if not any(map(lambda pkg: pkg.startswith("modin"), self.add_conda_packages)):
+        if not any(
+            map(lambda pkg: re.match(r"modin(\W|$)", pkg), self.add_conda_packages)
+        ):
```

Comment:
I just wonder if this would be shorter if formatted by `black`:
```suggestion
        if not any(re.match(r"modin(\W|$)", p) for p in self.add_conda_packages):
```
but that's really minor